{
    "type": "bundle",
    "id": "bundle--76d04b72-2fd2-4576-8bdd-53b75f1e049f",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--ab743b3a-4c5d-469b-af8f-e538885df65c",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2024-03-09T00:26:17.956596Z",
            "modified": "2024-03-09T00:26:17.956596Z",
            "name": "CVE-2024-23235",
            "description": "A race condition was addressed with additional validation. This issue is fixed in macOS Sonoma 14.4, visionOS 1.1, iOS 17.4 and iPadOS 17.4, watchOS 10.4, iOS 16.7.6 and iPadOS 16.7.6, tvOS 17.4. An app may be able to access user-sensitive data.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2024-23235"
                }
            ]
        }
    ]
}